From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Beyonder <kanup4m@rising>
Date: Wed, 30 Sep 2026 00:00:00 +0000
Subject: [PATCH] CallLogProvider: drop removed PREFERRED_DISPLAY_NAME

Rising CallLog.Calls has no PREFERRED_DISPLAY_NAME. Stop projecting
and restoring it; UUID backup coverage is unaffected.
---
diff --git a/src/com/android/calllogbackup/CallLogBackupAgent.java b/src/com/android/calllogbackup/CallLogBackupAgent.java
index 1111111..2222222 100644
--- a/src/com/android/calllogbackup/CallLogBackupAgent.java
+++ b/src/com/android/calllogbackup/CallLogBackupAgent.java
@@ -183,8 +183,7 @@ public class CallLogBackupAgent extends BackupAgent {
         CallLog.Calls.IS_PHONE_ACCOUNT_MIGRATION_PENDING,
         CallLog.Calls.IS_BUSINESS_CALL,
         CallLog.Calls.ASSERTED_DISPLAY_NAME,
-        CallLog.Calls.UUID,
-        Calls.PREFERRED_DISPLAY_NAME
+        CallLog.Calls.UUID
     };
 
     /**
@@ -768,8 +767,6 @@ public class CallLogBackupAgent extends BackupAgent {
         call.assertedDisplayName =
                 cursor.getString(cursor.getColumnIndex(CallLog.Calls.ASSERTED_DISPLAY_NAME));
         call.uuid = cursor.getString(cursor.getColumnIndex(CallLog.Calls.UUID));
-        call.preferredDisplayName = cursor.getString(cursor.getColumnIndex(
-                Calls.PREFERRED_DISPLAY_NAME));
         /*
          * Starting Android T, the ID of Telephony PhoneAccountHandle need to migrate from IccId
          * to SubId. Because the mapping between IccId and SubId in different devices is different,
-- 
2.55.0
